benchmarking/README.md

```diff
@@ -36,3 +36,78 @@ python examples/offline_inference_tt.py --measure_perf --max_seqs_in_batch 32 --
 - `--max_seqs_in_batch` (default: `32`):
   - **Maximum batch size** for inference, determining the number of prompts processed in parallel.
 
+### Online Benchmarking
+
+#### single user
+
+```bash
+python utils/prompt_client_cli.py \
+    --num_prompts 32 \
+    --batch_size 1 \
+    --tokenizer_model meta-llama/Llama-3.1-70B-Instruct \
+    --max_prompt_length 128 \
+    --input_seq_len 128 \
+    --output_seq_len 128 \
+    --template chat_template \
+    --dataset random
+```
+
+#### using vllm/benchmarking/benchmark_serving.py
+Within the Docker container, use the benchmark_serving.patch file:
+```
+cd ~/app/src
+python run_vllm_api_server.py
+```
+This simply stops the benchmarking script from sending the `best_of` arg which is not supported and causes issues.
```

Comment:
Is there an issue open for this in vLLM? Should it be something we open and get a permanent fix for?

Reply:
It's an issue with how we support parameters in our vLLM fork. Just filed https://github.com/tenstorrent/vllm/issues/44 to support this.